modules/social_features/social_user/social_user.module

```diff
@@ -675,31 +675,40 @@ function social_user_ensure_help_text_fieldset(&$form) {
  * displays the one with the lowest weight (highest priority).
  */
 function social_user_user_format_name_alter(&$name, AccountInterface $account) {
-  // We always add the username as fallback suggestion.
-  $suggestions = [
-    'username' => [
-      'name' => $account->getAccountName(),
-      'weight' => PHP_INT_MAX,
-    ],
-  ];
+  $suggested_names = &drupal_static(__FUNCTION__, []);
+
+  if (empty($suggested_names[$account->id()])) {
```

Comment:
This will be a future PHPStan rule that we enable, we can prefer `isset` here over `empty` which is slightly more specific (and thus performant).

```suggestion
  if (!isset($suggested_names[$account->id()])) {
```